### PR TITLE
Add option for dropdownButton

### PR DESCRIPTION
**Pull request by rifton007**
This pull add option "dropdown-menu-right".
To default, the dropdownButton is aligned to left.
The align property if it is set, waits for the 'right' or 'left' values.

### CODE REVIEW

File: src/View/Helper/FormHelper.php [hunk 1]
```diff
@@ -665,12 +665,14 @@ public function buttonToolbar(array $buttonGroups, array $options = array()) {
      * @return string A HTML string containing the button dropdown.
      */
     public function dropdownButton($title, array $menu = [], array $options = []) {
+        $align = (isset($options['align'])) ? $options['align'] : 'left';
```
Comment (Holt59) [inline src/View/Helper/FormHelper.php]: Could you use the "CakePHP way" of adding options here?

```php
$options += [
    'align' => 'left',
    'type' => false,
    'data-toggle' => 'dropdown'
];
$align = $options['align'];
unset($options['align']);
```

**Comment by Holt59** [comment]
@rifton007 Actually, this should go to `HtmlHelper::dropdown`, and the `FormHelper::dropdown` should only forward the `align` option to the `HtmlHelper::dropdown` (or something similar), could you modify that?

**Comment by rifton007** [comment]
I tried, but I do not understand how to do it.
Could you explain to me how to do it and I would do it with pleasure.
Sorry to not be able to do better.

**Comment by Holt59** [comment]
@rifton007 I'll merge this pull request this week-end to a separate branch and make the change to the test cases before merging it to the `master` branch.

**Comment by rifton007** [comment]
Thank you Holt 59, I will watch your changes to better understand.
Sorry still not to be able to do it myself.

**Comment by Holt59** [comment]
I have merged this PR to the [pr129](https://github.com/Holt59/cakephp3-bootstrap-helpers/tree/pr129) branch. I will probably move the part to the Html helper, and add the necessary tests.

**Comment by Holt59** [comment]
I have fixed issue with the PR and the tests:

- I have moved the `align` option to `HtmlHelper::dropdown` (it probably belongs here rather than in the `FormHelper`);
- I have forwarded the `align` option from `FormHelper::dropdown` to `HtmlHelper::dropdown` (and in the same time added a way to easily add new forwarding options);
- I have fix the tests, all that was needed was to add the `dropdown-menu-left` class to the generated dropdown (since it is now added by default).

Thanks again for the initial PR!

**Comment by rifton007** [comment]
Thank you @Holt59
Thanks to you for this plugin and all the work you do

**Pull request merged**